include/solana/account.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "publickey.hpp"
+#include "request.hpp"
+#include "types.hpp"
+
+namespace sol {
+    struct AccountMeta
```

Comment:
account meta is only used to declare instructions invocation, part of sealevel runtime spec
account info is only used to encode rpc responses

wouldn't put theminto the same file, very different parts of solana